daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg

```diff
@@ -3,31 +3,15 @@
 # See stub.cfg for details on how the options, packages, pre and post files fit
 # together.
 
-# 1
-%post --nochroot
-# copy google supplied RPM packages to image.
-
-mkdir /mnt/sysimage/tmp/gce-rpm-install
-isodir=/mnt/isodir
-cp $isodir/*.rpm /mnt/sysimage/tmp/gce-rpm-install
-
-%end
-
 # 2
 %post
 # Install EPEL for DKMS support.
 rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
 
 # Install google RHUI client software.
-pkgs="google-rhui-client"
-for package in ${pkgs}; do
-    rpm="/tmp/gce-rpm-install/${package}.rpm"
-    [ -f ${rpm} ] || continue
-    yum -y install ${rpm}
-    KEYS="$(rpm -qlp ${rpm} | grep ^/etc/pki/rpm-gpg/)"
-    rpm --import ${KEYS}
-done
-rm -rf /tmp/gce-rpm-install
+yum -y install google-rhui-client-rhel6
```

Comment:
Me gusta.

Reply:
Also, is this an issue? I thought there were issues with exposing the Google RHUI clients publicly? Is the yum repo public?

Reply:
Its fine for now.